rest/data/project_settings.go

```diff
@@ -184,7 +190,11 @@ func SaveProjectSettingsForSection(ctx context.Context, projectId string, change
 		}
 		modified = true
 	case model.ProjectPageGithubAndCQSection:
-		if err = handleGithubConflicts(mergedProjectRef, "Toggling GitHub features"); err != nil {
+		conflictProjectRef := mergedProjectRef
+		conflictProjectRef.Owner = mergedBeforeRef.Owner
```

Comment:
Can we just add these directly to mergedProjectRef? The backend only updates fields that are on that page so it wouldn't cause a problem (since mergedProjectRef is a pointer anyway I'm not sure this isn't already modifying it)